src/main/java/com/example/application/ui/horizontal/library/DocumentsView.java

```diff
@@ -70,16 +73,21 @@ public DocumentsView(PageService pageService, UserService userService, DocumentS
         pageEntity = pageService.findPageById(14);
         pageTitle = new H1(pageEntity.getTitle());
         pageText = new Paragraph(pageEntity.getContent());
-
+        GetUserController getUserController = new GetUserController();
+        String username = getUserController.getUsername();
+        UserEntity userEntity = userService.findByUsername(username);
+        RoleEntity roleEntity = userEntity.getRole();
+        role = roleEntity.getRoleId();
         breadcrumbs = new Breadcrumbs();
         breadcrumbs.add(new Breadcrumb("Home"), new Breadcrumb("Bibliothek"), new Breadcrumb(pageEntity.getTitle()));
 
         setData();
         //userEntity = pageEntity.getUser();
-
-        initializeRadioButtonsForKeyword();
+        if(role == 1) {
+            initializeRadioButtonsForKeyword();
+            initializeUploadContainer();
+        }
         initializePdfsManager();
```

Comment:
the pdfsManager should be initialized before initializing the uploadContainer, otherwise, the uploader would throw a NullPointerException, as it didn't find the pdfsManager. 
![image](https://user-images.githubusercontent.com/47325924/106489805-7170ad00-64b5-11eb-8b14-608b78947df0.png)

Reply:
Kann man den dann nicht darüber packen?

Reply:
ja das sollte funktionieren

Reply:
Dann tausche ich das eben